src/schema/me/conversation/index.js

```diff
@@ -199,8 +199,25 @@ export const ConversationFields = {
   last_message: {
     type: GraphQLString,
     description: "This is a snippet of text from the last message.",
-    resolve: conversation => {
-      return get(conversation, "_embedded.last_message.snippet")
+    resolve: (
+      conversation,
+      options,
+      request,
+      { rootValue: { conversationMessagesLoader } }
+    ) => {
+      // return ""
+
+      return conversationMessagesLoader({
+        conversation_id: conversation.id,
+        "expand[]": "deliveries",
+      }).then(({ message_details }) => {
+        const relevantDelivery = message_details.find(
+          m =>
+            m.from_email === conversation.from_email ||
```

Comment:
need to double check if we actually get `m.from_email` we may want to do `m.from_email_address_conversation.email_address` or something like that.